CMakeLists.txt

```diff
@@ -128,9 +128,44 @@ if (TRIBOL_STYLE_CI_ONLY)
 endif()
 
 #------------------------------------------------------------------------------
-# Add TPLs and source directories
+# Add TPLs
 #------------------------------------------------------------------------------
 include(cmake/SetupThirdParty.cmake)
+
```

Comment:
@chapman39 This block will have to be added to the `SetupSeracThirdParty.cmake` file.  Would it be better to put this in a file in the `cmake` folder (that's how axom does it)?

Reply:
Nevermind, probably not needed!